### PR TITLE
Dpp 1273 elevate qualification on the important metrics to debug

### PR DESCRIPTION
**Pull request by atriantafyllos-da**
[DPP-1273](https://digitalasset.atlassian.net/browse/DPP-1273)

### CODE REVIEW

**Comment by atriantafyllos-da** [comment]
The first commit converts the qualification to a `String` in order to be able to parse it as annotation.
The second elevates the qualification of the most important metrics.

File: ledger/metrics/src/main/scala/com/daml/metrics/MetricHandle.scala [hunk 1]
```diff
@@ -136,19 +136,10 @@ object MetricHandle {
...
-    case object Traffic extends MetricQualification
-    case object Errors extends MetricQualification
-    case object Saturation extends MetricQualification
-    case object Debug extends MetricQualification
-  }
-
   case class Tag(
       summary: String,
       description: String,
-      qualification: MetricQualification,
```
Comment (mziolekda) [inline ledger/metrics/src/main/scala/com/daml/metrics/MetricHandle.scala:151]: I don't understand why it is necessary. What's wrong with `qualification.toString` wherever you need it?

**Comment by atriantafyllos-da** [comment]
Reverted the conversion of qualification to String

File: ledger/metrics/src/main/scala/com/daml/metrics/ServicesMetrics.scala [hunk 2]
```diff
@@ -179,6 +179,14 @@ class ServicesMetrics(override val prefix: MetricName, override val registry: Me
...
     val writeOperationForDocs: Timer = Timer(prefix :+ "<operation>", null)
 
+    @MetricDoc.Tag(
+      summary = "The time to execute a write service operation.",
+      description = """The write service is an internal interface for changing the state through
+                      |the synchronization services. The methods in this interface are all methods
+                      |that are supported uniformly across all ledger implementations. This metric
+                      |exposes the time needed to execute each operation.""",
+      qualification = Traffic,
+    )
```
Comment (mziolekda) [inline ledger/metrics/src/main/scala/com/daml/metrics/ServicesMetrics.scala:189]: 1. The text above seems to be the general text that accompanies all <operations>. It needs to be slightly more adopted to the `submit_transaction`
2. Actually it is only the _count portion of it that is a Traffic metric, I would suggest to do 
`val submitOperationForDocs: Timer = Timer(prefix :+ "submit_transaction_count", null)`

Comment (atriantafyllos-da) [reply ledger/metrics/src/main/scala/com/daml/metrics/ServicesMetrics.scala:189]: i.e. to be more specific regarding which component of the metric we are using (mean, count)

File: ledger/metrics/src/main/scala/com/daml/metrics/ServicesMetrics.scala [hunk 3]
```diff
@@ -179,6 +179,16 @@ class ServicesMetrics(override val prefix: MetricName, override val registry: Me
...
 
+    @MetricDoc.Tag(
+      summary = "The number of submitted transactions by the write service.",
+      description = """The write service is an internal interface for changing the state through
+                      |the synchronization services. The methods in this interface are all methods
+                      |that are supported uniformly across all ledger implementations. This metric
+                      |exposes the total number of the sumbitted transactions.""",
+      qualification = Traffic,
+    )
+    val submitOperationForDocs: Timer = Timer(prefix :+ "submit_transaction" :+ "_count", null)
```
Comment (mziolekda) [inline ledger/metrics/src/main/scala/com/daml/metrics/ServicesMetrics.scala:190]:
```suggestion
    val submitOperationForDocs: Timer = Timer(prefix :+ "submit_transaction" :+ "count", null)
```

File: ledger/metrics/src/main/scala/com/daml/metrics/ServicesMetrics.scala (hunk 2 above)
Comment (mziolekda) [reply ledger/metrics/src/main/scala/com/daml/metrics/ServicesMetrics.scala:189]: I thin it only affects timers used for Traffic

**Review by mziolekda — APPROVED** [review]

**Pull request merged**